packages/benchmarks/src/textToDriver/loadBraintrustDatasets.ts

```diff
@@ -115,3 +126,23 @@ export async function loadBraintrustDbDocuments({
 
   return dbDocuments;
 }
+
+/**
+  Load the evaluation cases for the text-to-driver task from Braintrust.
+ */
+export async function loadBraintrustEvalCases({
+  apiKey,
+  projectName,
+  datasetName = "text-to-query-results",
```

Comment:
```suggestion
  datasetName,
```

Wondering if we should remove this default in favor of always being explicit about the dataset name?

Reply:
reasonable i think